packages/stream_video/example/lib/main.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flutter/material.dart';
+import 'package:stream_video/stream_video.dart';
+
+Future<void> main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+
+  StreamVideo.create(
+    'api_token',
+    user: User.guest(userId: 'user_id'),
+  );
+}
```

Comment:
Is this an example for pub?

Reply:
yes, we had it before just like this (probably just for the pub scoring) then it got removed and scoring with it. So now it's back :)

Reply:
This is fine for now but let's add these things in separate PRs so we don't confuse stuff.